.gitmodules

```diff
@@ -4,3 +4,6 @@
 [submodule "ThirdParty/NVorbis"]
 	path = ThirdParty/NVorbis
 url=https://github.com/MrHelmut/NVorbis.git
```

Comment:
This makes me think that we should probably fork our own https://github.com/ioctlLR/NVorbis or point back to it.

The critical PR that was pending and resulted in pointing to my fork ultimately got merged upstream. NVorbis haven't been updated since then.